sycl/test/optional_kernel_features/fp-accuracy.cpp

```diff
@@ -0,0 +1,138 @@
+// RUN: %clangxx %s -o %test.bc -ffp-accuracy=high:sin,sqrt -ffp-accuracy=medium:cos -ffp-accuracy=low:tan -ffp-accuracy=cuda:exp,acos -ffp-accuracy=sycl:log,asin  -fno-math-errno  -fsycl -fsycl-device-only
```

Comment:
Is it possible to test if the aspects got correctly propagated to the calling functions?

Thanks